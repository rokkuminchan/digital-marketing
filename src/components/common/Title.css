.title {
  color: #3c64b1;
  font-weight: bold;
  text-align: center;
  padding: 4rem 3rem 0;
}

/*Iphone (480 x 640)*/
@media screen and (min-width: 320px) {
  .title {
    font-size: 2.5em;
  }
}

/*Ipad vertical(768 x 1024)*/
@media screen and (min-width: 768px) {
  .title {
    font-size: 3.5em;
  }
}

/*desktop*/
@media screen and (min-width: 1024px) {
  .title {
    font-size: 3.8em;
  }
}
