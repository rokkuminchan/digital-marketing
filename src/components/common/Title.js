import React from "react";
import "./Title.css";

export default function Title({data}) {
  return (
    <div className="title">
      <h3 className="title">{data}</h3>
    </div>
  );
}
