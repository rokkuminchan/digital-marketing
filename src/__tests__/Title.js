import React from "react";
import renderer from "react-test-renderer";
import Title from "../components/common/Title";

describe("Title", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Title data="Default Starter" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
