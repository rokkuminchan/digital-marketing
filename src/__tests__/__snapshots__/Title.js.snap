// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<div
  className="title"
>
  <h3
    className="title"
  >
    Default Starter
  </h3>
</div>
`;
